src/js/yeldaWebchatInjector.js

```diff
@@ -365,22 +365,26 @@ class YeldaChat {
   }
 
   /**
-   * Handles communcation between parent window and iframe, mainly for open and closing the chat
+   * Handles communication between parent window and iframe, mainly for open and closing the chat
    * handles also the chat bubble style
    * @param {event} event
    */
   messageListener(event) {
-    if (event.data === 'closeChat' || event.message === 'closeChat') {
+    if (!event.data) {
+      return
+    }
+
+    if (event.data === 'closeChat') {
```

Comment:
1. i saw your big exchange with Naomi, but i am not sure i got everything : so it's ok to only check event.data and no more event.message ?
2. same feedback than the other PR : let's start to use const values instead of harcoded strings like "closeChat" ?